cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/elements/EnergyConsumerEq.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.export.elements;
+
+import com.powsybl.cgmes.model.CgmesNames;
+import com.powsybl.iidm.network.extensions.LoadDetail;
+
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+
+import static com.powsybl.cgmes.model.CgmesNamespace.RDF_NAMESPACE;
+
+/**
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+public final class EnergyConsumerEq {
+
+    public static void write(String id, String loadName, LoadDetail loadDetail, String equipmentContainer, String cimNamespace, XMLStreamWriter writer) throws XMLStreamException {
+        writer.writeStartElement(cimNamespace, loadClassName(loadDetail));
+        writer.writeAttribute(RDF_NAMESPACE, CgmesNames.ID, id);
+        writer.writeStartElement(cimNamespace, CgmesNames.NAME);
+        writer.writeCharacters(loadName);
+        writer.writeEndElement();
+        writer.writeEmptyElement(cimNamespace, "Equipment.EquipmentContainer");
+        writer.writeAttribute(RDF_NAMESPACE, CgmesNames.RESOURCE, "#" + equipmentContainer);
+        writer.writeEndElement();
+    }
+
+    private static String loadClassName(LoadDetail loadDetail) {
```

Comment:
I think that conform load means that the variable part is not equal to zero. A non conform load means that the variable part is equal to zero.

Reply:
Yes, the code follows your interpretation. To avoid duplication, this method will be moved to the module CgmesExportUtil, as it is required for EQ export and also for SSH export